### PR TITLE
PhpdocAnnotationWithoutDotFixer - handle ellipsis

### LINKED ISSUE #2027: PhpdocAnnotationWithoutDotFixer removes '...' step by step
```php
/**
 * @param string $foo Foo, e. g. "bar", "baz", ... 
 */
```

`PhpdocAnnotationWithoutDotFixer` removes the '...' one by one. So you have to run the fixer 3 times to remove all dots.

### PR DESCRIPTION
**Pull request by keradus**
Fixes #2027

### CODE REVIEW

**Comment by keradus** [comment]
cc @gharlan

File: Symfony/CS/Tests/Fixer/Symfony/PhpdocAnnotationWithoutDotFixerTest.php [hunk 1]
```diff
@@ -37,10 +37,12 @@ public function provideFixCases()
      *
      * Description.
      *
-     * @param string $str   Some string
-     * @param bool   $isStr Is it a string?
-     * @param int    $int   Some multiline
-     *                      description. With many dots
+     * @param string $str        Some string
+     * @param string $ellipsis1  Ellipsis is this: ...
+     * @param string $ellipsis2  Ellipsis is this: 。。。
```
Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/PhpdocAnnotationWithoutDotFixerTest.php]: lol, who uses that?

**Comment by GrahamCampbell** [comment]
Should we add a test for the single character version too?

**Comment by keradus** [comment]
Like this: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2050/files#diff-4e353d9e86349c0daec425c628ad5688R57 ?

**Comment by GrahamCampbell** [comment]
Yeh, looks good.

**Pull request closed without merging**